srcs/expansion/command_substitution.c

```diff
@@ -67,6 +68,8 @@ static void					add_substitution(t_strlist **strlist_addr
 	extern char			**environ;
 	char				**environ_backup;
```

Comment:
same here as in #134 , no need for extern environ + backup, just one env set to get_variable_for_execution